frontEnd/tickets-app/src/components/Main/Main.js

```diff
@@ -1,17 +1,40 @@
-import React from "react";
-import SignIn from "../../pages/SignIn/SignIn";
-import { Route, Routes } from "react-router-dom";
-import Footer from "../Footer/Footer";
-import { useSelector } from "react-redux";
-import { getUser } from "../../store/user/selectors/selectors";
+import React, { useEffect } from "react";
+import LogIn from "../../pages/LogIn/LogIn";
+import { Route, Routes, useParams } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { getUser } from "../../store/user/selectors";
+import * as actions from "../../store/user/actions";
+import "./Main.scss";
+import SignUp from "../../pages/SignUp/SignUp";
+import MainPage from "../../pages/MainPage/MainPage";
+import NavBar from "../NavBar/NavBar";
+import Category from "../../pages/Category/Category";
+import InfoFilm from "../../pages/InfoFilm/InfoFilm";
 
 const Main = () => {
   const user = useSelector(getUser);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(actions.local());
+  }, []);
+  const param = useParams();
+
   return (
     <main>
+      <NavBar />
       <Routes>
-        <Route path="/" element={<Footer />} />
-        {!user && <Route path="registration" element={<SignIn></SignIn>} />}
+        {!user && (
+          <>
+            <Route path="login" element={<LogIn />} />
+            <Route path="signup" element={<SignUp />} />
+          </>
+        )}
+        <Route
+          path="category/:category"
+          element={<Category category={param.category} />}
```

Comment:
you should take a category from `const params = useParams()` inside a `Category` component